packages/riipen-ui/src/components/Tab.test.jsx

```diff
@@ -0,0 +1,363 @@
+        },
+        {
+          breakpoint: "none",
+          orientation: "horizontal",
+          argument: "breakpoint",
+          expected: false
+        }
+      ];
+
+      tests.forEach(test => {
+        it(`sets ${test.expected} class name with custom breakpoint and orientation prop values`, () => {
+          props.breakpoint = test.breakpoint;
+          props.orientation = test.orientation;
+
+          const wrapper = mount(<Tab {...props} value />);
+
+          expect(
+            wrapper
+              .find("Tab")
+              .childAt(0)
+              .hasClass(test.argument)
+          ).toEqual(test.expected);
+        });
+      });
+    });
+
+    it("gives an error when given an invalid breakpoint", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+      const breakpoint = "xs";
+
+      mount(<Tab breakpoint={breakpoint} value />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<Tab classes={classes} value />);
+
+      expect(
+        wrapper
+          .find("Tab")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("color prop", () => {
+    it("sets custom color", () => {
+      const color = "secondary";
+
+      const wrapper = mount(<Tab color={color} value />);
+
+      expect(
+        wrapper
+          .find("Tab")
+          .childAt(0)
+          .hasClass(color)
+      ).toEqual(true);
+    });
+
+    it("gives an error when given an invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+      const color = "tertiary";
+
+      mount(<Tab color={color} value />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("disabled prop", () => {
+    it("sets disabled class name with valid custom disabled", () => {
+      const disabled = true;
+
+      const wrapper = mount(<Tab disabled={disabled} value />);
+
+      expect(
+        wrapper
+          .find("Tab")
+          .childAt(0)
+          .hasClass("disabled")
+      ).toEqual(true);
+    });
+  });
+
+  describe("fullWidth prop", () => {
+    it("sets fullWidth class name with valid custom fullWidth", () => {
+      const fullWidth = true;
+
+      const wrapper = mount(<Tab fullWidth={fullWidth} value />);
+
+      expect(
+        wrapper
+          .find("Tab")
+          .childAt(0)
+          .hasClass("full-width")
+      ).toEqual(true);
+    });
+  });
+
+  describe("icon prop", () => {
+    it("sets icon class name with valid custom icon", () => {
+      const wrapper = mount(<Tab icon={icon(faPhone)} value />);
+
+      expect(
+        wrapper
+          .find("Tab")
+          .childAt(0)
+          .childAt(0)
+          .hasClass("icon")
+      ).toEqual(true);
+    });
+  });
+
+  describe("label prop", () => {
```

Comment:
I would maybe add a test to make sure it renders the label value as well.

Reply:
Added